add feedback message after saving

josihoppe opened the pull request:
- changed order of footer buttons according to codepen (https://codepen.io/michaelbr/pen/QwjEXjK) in step2
- added feedback message toast and saveSpinner for saving
- adjusted layout according to the codepen stylings

josihoppe committed:
changed order of footer buttons according to codepen

josihoppe committed:
added functionality to feedback message

josihoppe committed:
fixed/reverted position of buttons

josihoppe committed:
moved toast styling to scss and fixed naming issue

josihoppe committed:
moved save button closer to right edge

josihoppe committed:
applied stylings to toast text and body

Bachibouzouk commented on app/static/scss/components/_footer.scss:
```diff
@@ -58,6 +58,32 @@
 	}
 }
```
The compiled css is style useful, as serving css is faster than compiling scss and then serve the css :) The first commits were right to add to css. We also add it to the scss because @bmlancien works with scss, which then generates css that we add to main.css

josihoppe replied:
Hm. i moved the style back to the css. I had put it in the scss, because I couldnt see the changes that I made, while it was only in the css...

Bachibouzouk commented on app/templates/scenario/scenario_step2.html:
```diff
@@ -282,28 +295,64 @@ <h3>{{ group_names|get_item:group_name|title }}</h3>
+                    <path d="m65.691 35.762-21.68 21.668-9.6992-9.6992c-1.25-1.25-3.2891-1.25-4.5391 0s-1.25 3.2891 0 4.5391l11.969 11.969c0.62891 0.62891 1.4492 0.94141 2.2695 0.94141 0.82031 0 1.6406-0.30859 2.2695-0.94141l23.949-23.938c1.25-1.25 1.25-3.2891 0-4.5391-1.2617-1.2617-3.2891-1.2617-4.5391 0z"/>
+                    <path d="m50 10c-22.059 0-40 17.941-40 40s17.941 40 40 40 40-17.941 40-40-17.941-40-40-40zm0 73.578c-18.512 0-33.578-15.059-33.578-33.578s15.066-33.578 33.578-33.578 33.578 15.059 33.578 33.578-15.066 33.578-33.578 33.578z"/>
+                </svg>
+                <span>Saved successfully<span>
```
```suggestion
                <span>Saved successfully</span>
```

Bachibouzouk replied:
@josihoppe - I let you commit the suggestion, that way I make sure you don't get a problem if you read this after you already commited the compiled main.css

Bachibouzouk reviewed (changes requested):
After you add main.css and move the button "To simulation" in step 1 and 3 as well you we can merge the PR

josihoppe committed:
add missing /

Co-authored-by: Pierre Francois <pierre-francois.duc@rl-institut.de>

josihoppe committed:
moved simulation button to center in step 1 and 3

josihoppe committed:
readded style to main.css

Bachibouzouk committed:
Only display Go to simulation when a simulation exists

Also infer the step number of the simulation page from the max_step
value rather than hardcoding it

Bachibouzouk committed:
Update main.css from compiled scss

Bachibouzouk committed:
Fix failing test

Bachibouzouk reviewed (approved):
Thanks @josihoppe for this!
